.changeset/large-news-grin.md

```diff
@@ -0,0 +1,7 @@
+---
+"@aws-amplify/ui-react-core": patch
+"@aws-amplify/ui-react-native": patch
+"@aws-amplify/ui-react": patch
```

Comment:
Does the `ui-react` package need to be bumped?

Reply:
Think so since since react-core is getting bumped?

Reply:
Let's not add `ui-react` package here, since it wasn't changed. It will still get bumped bc `react-core` was changed but it won't have below changeset.